packages/app/src/commonTypes.ts

```diff
@@ -64,6 +72,7 @@ export type SavedSearch = z.infer<typeof SavedSearchSchema>;
 export const SavedChartConfigSchema = z.any();
 
 export const TileSchema = z.object({
+  name: z.string().optional(),
```

Comment:
Is this correct? `name` should be within `config` field, right?

Reply:
correct, also removed `commonTypes.ts` from app package since I import everything from common utils now